ui/serializers.py

```diff
@@ -135,6 +137,14 @@ def validate_view_lists(self, value):
         """
         return validate_moira_lists(value)
 
+    def get_cloudfront_url(self, obj):
+        """Get cloudfront_url"""
+        video_file = obj.videofile_set.filter(encoding=EncodingNames.HLS).first()
+        if obj.collection.allow_share_openedx and video_file:
```

Comment:
I think this is supposed to be viewable only by the collection owner (and maybe people on the admin moira lists)?

Reply:
I think `SimpleVideoSerializer` also needs to be modified, if I click "Share" for a video from the collections page, the cloudfront url field doesn't appear:

![Screen Shot 2019-06-17 at 4 52 07 PM](https://user-images.githubusercontent.com/187676/59636031-8d78dc80-9120-11e9-8c53-4e4e0f0187ae.png)